### PR TITLE
Update closed-functionality.md - modify examples for comments on clos…

### PR DESCRIPTION
**Pull request by pday1**
…ed functionality

From q4 & 5 on survey
https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-closed-more-to-review/results

### CODE REVIEW

**Comment by maryjom** [comment]
@pday1 I see that your goal in this PR was to simplify the list of examples. However, I think people were OK with this list having a bit more detail (explanation) than in the term definition's list. Especially the eBook example, as Shadi had asked for me to leave that as it was in the 2013 WCAG2ICT.

**Comment by maryjom** [comment]
After looking at the two PRs you did, I think I understand what was done. The more detailed examples were moved into the definition, and the basic version of the examples are in the Closed Functionality intro section - swapping their location.  Am I getting that right?

File: closed-functionality.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ To the extent the ICT is closed, following the WCAG success criteria by themselv
...
-*   entertainment technologies including smart TVs, set-top boxes, smart displays, smart speakers, smart watches, and tablets
-*   an ebook reader or standalone ebook software that allows assistive technologies to access all of the user interface controls of the ebook program (open functionality) but does not allow the assistive technologies to access the actual content of book (closed functionality).
-*   an operating system that requires the user to provide log in credentials before it allows any assistive technologies to be loaded. The log-in portion would be closed functionality.
-*   other technology devices, such as printers, displays, and iOt devices</div>
+* self-service transaction machines or kiosks (e.g. retail self-checkout, point of sales (POS) terminals, and Automated Teller Machines (ATMs))
+* telephony devices (e.g. IP phones, feature phones, and smartphones)
+* entertainment technologies (e.g. smart TV, set-top box, smart watches)
+* ebook reader
+* computer that is locked down due to a policy so that users may not adjust settings or install software
+* other technology devices (e.g. printers, displays, and internet of things (iOt) devices).
```
Comment (maryjom) [inline closed-functionality.md:29]:
```suggestion
* other technology devices (e.g. printers, displays, and Internet of Things (IoT) devices).
```

**Review by bruce-usab — APPROVED** [review]
I agree with that this list of examples is an improvement.

**Pull request merged**